compiler/rewrite-rules/early_opt.rules

```diff
@@ -12,6 +12,12 @@
 (op_func_call {"hrtime"} (op_false)) => (op_func_call {"_hrtime_array"})
 (op_func_call {"hrtime"} (op_true))  => (op_func_call {"_hrtime_int"})
 
+(op_func_call {"curl_setopt"} curl_handle option:(op_int_const {"210000"}) cb)
```

Comment:
Allows redirecting the call of the curl_setopt() function to the specified _curl_setopt_fn_header_write() function if the option with the number 210000 is passed as a parameter